### PR TITLE
Feature/dsn 4063

### PR DESCRIPTION
**Pull request by anis790615**

### CODE REVIEW

File: src/Jobs/Feeds/Feed.php [hunk 1]
```diff
@@ -35,22 +37,21 @@ abstract class Feed implements ShouldQueue, ShouldBeUnique
     /** @var Store */
     protected $store;
 
-    /** @var S3Client */
-    protected $s3Client;
-
     /** @var string */
     protected $feedsBucket = '';
```
Comment (vincentvanhoven) [inline src/Jobs/Feeds/Feed.php:41]: This can be removed

File: src/Jobs/Feeds/Feed.php [hunk 2]
```diff
@@ -99,13 +93,11 @@ public function handle(ProductRepository $productRepository, S3ClientInterface $
...
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Builder
-     */
-    protected function getProductQuery()
+    protected function getProductQuery(): Builder
     {
         return $this->productRepository->newQuery()
             ->where('is_for_sale', 1)
+            ->where('id', '<', 59)
```
Comment (vincentvanhoven) [inline src/Jobs/Feeds/Feed.php:100]: Careful now 😉 
Can you remove this line?

**Review by vincentvanhoven — APPROVED** [review]
Excellent PR. There is two small comments, can be merged after.

File: src/Jobs/Feeds/Feed.php (hunk 2 above)
Comment (anis790615) [reply src/Jobs/Feeds/Feed.php:100]: Oh, forgot about it.

**Pull request merged**